site/en/r2/tutorials/load_data/csv.ipynb

```diff
@@ -0,0 +1,933 @@
+{
```

Comment:
<p>Can you also print the real label?</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/487/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='61'/>